ai_edge_torch/generative/utilities/stable_diffusion_loader.py

```diff
@@ -698,6 +698,7 @@ def load(
 
     attention_config = layers_config.AttentionConfig(
         num_heads=config.transformer_num_attention_heads,
+        head_dim=config.block_out_channels[0] // config.transformer_num_attention_heads,
```

Comment:
This change will need @yichunk to review

Reply:
For context: I'm just using the way head_dim was previously defined [here](https://github.com/google-ai-edge/ai-edge-torch/blob/9fa04010af0aabfb1b97da1675202432155d138b/ai_edge_torch/generative/layers/model_config.py#L157-L158)

Reply:
Same as config defined in `diffusion.py`. The head dim will be different for each block.